go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/rancher/rke2
 
-go 1.20
+go 1.21
+
+toolchain go1.22.0
```

Comment:
```suggestion
go 1.21
```

Reply:
Remove the `toolchain` line and then run the tidy with `GOTOOLCHAIN=$(. ./scripts/version.sh &>/dev/null; echo $VERSION_GOLANG) go mod tidy` and it will prevent go1.22 from adding this unwanted option. We're still building with 1.21 so there is no need for 1.22 specific changes on this branch.